doc/content/3.documentation/1.concepts/1.messages.md

```diff
@@ -90,14 +90,15 @@ A common mistake when engineers are new to messaging is to create a base class f
 | EntityName                  | The exchange or topic name                                                    |
 | ExcludeFromTopology         | Don't create an exchange or topic unless it is directly consumed or published |
 | ExcludeFromImplementedTypes | Don't create a middleware filter for the message type                         |
+| MessageUrn                  | The message urn                                                               |
```

Comment:
@phatboyg Would it benefit the docs if I add links here to the examples of the attributes?

Reply:
Yeah, that would be helpful sure!